### PR TITLE
static card memory pattern for more promo cards

### PR DESCRIPTION
**Pull request by bafolts**
More promo cards modified to use static memory pattern for properties.

### CODE REVIEW

File: src/Player.ts [hunk 1]
```diff
@@ -2142,10 +2142,7 @@ export class Player implements ISerializable<SerializedPlayer> {
         }
       }
       if (player.corporationCard instanceof PharmacyUnion) {
-        if (d.corporationCard.isDisabled === true) {
-          player.corporationCard.tags = [];
```
Comment (kberg) [inline src/Player.ts:2146]: Did you intend to remove this line?

File: src/cards/promo/Philares.ts [hunk 2]
```diff
@@ -6,128 +6,132 @@ import {ISpace} from '../../boards/ISpace';
...
-    public startingMegaCredits: number = 47;
-    public cardType = CardType.CORPORATION;
+export class Philares extends Card implements CorporationCard {
+  constructor() {
+    super({
+      cardType: CardType.CORPORATION,
+      name: CardName.PHILARES,
+      tags: [Tags.BUILDING],
+      startingMegaCredits: 47,
+      initialActionText: 'Place a greenery tile and raise the oxygen 1 step',
```
Comment (kberg) [inline src/cards/promo/Philares.ts:24]: Warning: initialActionText is all about rendering. It should be part of metadata, to prevent commingling..

**Review by kberg — APPROVED** [review]

File: src/Player.ts (hunk 1 above)
Comment (bafolts) [reply src/Player.ts:2146]: The `tags` property is only an accessor. While changing `PharmacyUnion` typescript wouldn't compile here. This is handled within `PharmacyUnion` now. When `this.isDisabled` is true the tags returns `[]`.

File: src/cards/promo/Philares.ts (hunk 2 above)
Comment (bafolts) [reply src/cards/promo/Philares.ts:24]: That is a good point. This happens to be used as the title for a `PlayerInput` so has been left behind. Should get moved to metadata.

**Pull request merged**